packages/expo-modules-core/src/uuid/uuid.types.ts

```diff
@@ -0,0 +1,20 @@
+type UUID = string;
+
+/**
+ * Collection of utilities used for generating Universally Unique Identifiers.
+ */
+export type Uuid = {
```

Comment:
Having both `UUID` and `Uuid` types is really confusing. I would suggest using `UUID` here and just `string` as the other type.
```suggestion
export type UUID = {
```